db/migrate/20151130111755_description_value_hash.rb

```diff
@@ -0,0 +1,19 @@
+class DescriptionValueHash < Mongoid::Migration
+  def self.up
+    puts "Migrating description strings into value hashes, e.g."
+    puts "'some description' : { 'value' => 'some description' }"
+
+    ContentItem.all.each.with_index do |item, index|
+      print "." if (index % 100).zero?
+
+      description = item["description"]
+
+      unless description.is_a?(Hash)
+        item.description = description
+        item.save!(validate: false)
```

Comment:
If we don't want validations perhaps `update_column` would be better, since that won't adjust the `updated_at` either (which is a fallback for public updated at, I think).